storage/innobase/os/os0file.cc

```diff
@@ -8447,32 +8454,50 @@ void Encryption::random_value(byte *value) {
   my_rand_buffer(value, ENCRYPTION_KEY_LEN);
 }
 
-void Encryption::fill_key_name(char *key_name, uint key_id) {
+void Encryption::fill_key_name(char *key_name, uint key_id, const char *uuid) {
```

Comment:
General comment: it looks like we have bunch of function that can name "keyring' in the function name.  This one and get_tablespace_key() and other friends.

It is optional and upto you :)